packages/mdc-tab/foundation.js

```diff
@@ -84,18 +100,54 @@ class MDCTabFoundation extends MDCFoundation {
 
   /**
    * Activates the Tab
+   * @param {ClientRect=} previousTabIndicatorRect
    */
-  activate() {
+  activate(previousTabIndicatorRect) {
     // Early exit
     if (this.isActive()) {
       return;
     }
-    this.adapter_.registerEventHandler('transitionend', this.handleTransitionEnd_);
+
+    this.activateTab_();
+    this.activateIndicator_(previousTabIndicatorRect);
+  }
+
+  /**
+   * Activates the tab
+   * @private
+   */
+  activateTab_() {
+    this.adapter_.registerRootEventHandler('transitionend', this.handleRootTransitionEnd_);
     this.adapter_.addClass(cssClasses.ANIMATING_ACTIVATE);
     this.adapter_.addClass(cssClasses.ACTIVE);
     this.adapter_.setAttr(strings.ARIA_SELECTED, 'true');
   }
 
+  /**
+   * Activates the tab indicator
+   * @param {ClientRect=} previousTabIndicatorRect
+   * @private
+   */
+  activateIndicator_(previousTabIndicatorRect) {
+    // Early exit
+    if (!previousTabIndicatorRect || this.adapter_.indicatorHasClass(cssClasses.INDICATOR_ICON)) {
```

Comment:
Early exit if the indicator is an icon? Why don't icon indicators require activation? This is not intuitive